docs/blog/2017-07-19-creating-a-blog-with-gatsby/index.md

```diff
@@ -0,0 +1,486 @@
+
+## Creating the Blog Listing
+
+I won't go into quite as much detail for this section, because we've already done something very similar for our blog template! Look at us, we're pro Gatsby-ers at this point!
+
+Gatsby has a standard for "listing pages," and they're placed in the root of our filesystem we specified in `gatsby-source-filesystem`, e.g. `src/pages/index.js`. So create that file if it does not exist, and let's get it working! Additionally note that any JavaScript files (typically React components!) will get a corresponding static HTML file when built. For instance, if we create `src/pages/tags.js`, the path `http://localhost:8000/tags.html` will be available within the browser and the statically generated site.
+
+```javascript
+import React from 'react';
+import Link from 'gatsby-link';
+import Helmet from 'react-helmet';
+
+// import '../css/index.css'; // add some style if you want!
+
+export default function Index({
+  data
+}) {
+  const { edges: posts } = data.allMarkdownRemark;
+  return (
+    <div className="blog-posts">
+      {posts
+        .sort((a, b) => {
+          const getDate = post => new Date(post.node.frontmatter.date);
+
+          return getDate(a) - getDate(b);
+        })
+        .reverse() // newest first
+        .filter(post => post.node.frontmatter.title.length > 0)
+        .map(({ node: post }) => {
+          return (
+            <div className="blog-post-preview" key={post.id}>
+              <h1>
+                <Link to={post.frontmatter.path}>{post.frontmatter.title}</Link>
+              </h1>
+              <h2>{post.frontmatter.date}</h2>
+              <p>{post.excerpt}</p>
+            </div>
+          );
+        })}
+    </div>
+  );
+}
+
+export const pageQuery = graphql`
+  query IndexQuery {
+    allMarkdownRemark {
+      edges {
+        node {
+          excerpt(pruneLength: 250)
+          id
+          frontmatter {
+            title
+            date(formatString: "MMMM DD, YYYY")
+            path
+          }
+        }
+      }
+    }
+  }
+`;
+```
+
+OK! So we've followed a similar approach to our blog post template, so this should hopefully seem pretty familiar. Once more we're exporting `pageQuery` which contains a GraphQL query. Note that we're pulling a slightly different data set, specifically we are pulling an `excerpt` of 250 characters rather than the full HTML, as well as we are formatting the pulled date with a format string! GraphQL is awesome.
+
+The actual React component is fairly trivial, but one important note should be made. It's important that when linking to internal content, i.e. other blog links, that it's always advisable to use `gatsby-link`. This is for a variety of reasons, but one key one is that the `Link` component includes the ability to use a `pathPrefix.` This is useful if this blog will be hosted on something like Github Pages, or an otherwise non-root domain level, which are both very common patterns.
```

Comment:
"it's always advisable"

Make this stronger e.g. "you're required to use gatsby-link". Gatsby doesn't work if you don't use gatsby-link.